app/screens/index.tsx

```diff
@@ -1,16 +1,15 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import {DEFAULT_LOCALE, getTranslations} from '@i18n';
```

Comment:
Ok let's have this imports follow a convention

1. Import Libraries sorted alphabetically
2. Import from our code base sorted alphabetically
3. Import from the same path sorted alphabetically

Each block should have an empty line between them.

for example
```js
import {withManagedConfig} from '@mattermost/react-native-emm';
import React from 'react';
import {IntlProvider} from 'react-intl';
import {Platform, StyleProp, ViewStyle} from 'react-native';
import {gestureHandlerRootHOC} from 'react-native-gesture-handler';
import {Navigation, NavigationFunctionComponent} from 'react-native-navigation';

import {Screens} from '@constants';
import {DEFAULT_LOCALE, getTranslations} from '@i18n';

import AnotherComponentInThisFolder from './another_component';

Reply:
Corrected